src/cpu/aarch64/brgemm/jit_brdgmm_kernel.cpp

```diff
@@ -455,7 +468,16 @@ void jit_brdgmm_kernel_base_t::store_accumulators_apply_post_ops(
                     else
                         st1w(vmm.s, P_ALL_ONE / T_m, addr);
                     break;
-                case data_type::bf16: assert(!"unsupported data type\n"); break;
+                case data_type::bf16:
+                    if (mask_flag) {
+                        bfcvt(vmm.h, k_mask / T_m, vmm.s);
+                        st1h(vmm.s, k_mask / T_m, addr);
+                    } else {
+                        bfcvt(vmm.h, P_ALL_ONE / T_m, vmm.s);
+                        st1h(vmm.s, P_ALL_ONE / T_m, addr);
+                    }
+
```

Comment:
```suggestion
                    const auto mask = mask_flag ? k_mask/T_m : P_ALL_ONE/T_m;
                    bfcvt(vmm.h, mask, vmm.s);
                    st1h(vmm.s, mask, addr);

```

we can hoist the mask var for other dtype as well.